Practice2/src/styles/components/common-component.css

```diff
@@ -0,0 +1,15 @@
+.heading {
```

Comment:
Please create a common/ into components. And split common components to separate files. e.g: buttons, heading, media...

Reply:
fixed